distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -515,27 +515,6 @@ private Response ingestXmlWithHeaderMetacard() {
         .post(CSW_PATH.getUrl());
   }
 
-  @Test
-  public void testCswIngest() {
```

Comment:
:question: Should we remove all of the other tests you moved to core?

Reply:
Done